tox.ini

```diff
@@ -1,10 +1,11 @@
 [tox]
 envlist =
-    py27,py34,py35,py36,py37
+    py27,py34,py35,py36,py37,py38
 
 [testenv]
 deps =
     zope.testrunner
+    zope.interface>=5.0.0
     .[test]
```

Comment:
A modern spelling is to say `extras = test` instead of adding `.[test]` to `deps`.  Saves a bit of time by not installing the current package twice (once without extras, and once with).

Reply:
I don't know much about tox, but this was already there. Good to clean it up though.
Done.